changelog/_unreleased/2022-10-16-fix-fetch-association-in-bulk-edit.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Fix fetch association in bulk edit
+author: Stefan Poensgen
+author_email: mail@stefanpoensgen.de
+author_github: @stefanpoensgen
+---
+# Administration
+* Fixes fetching associations in bulk edit to avoid infinite loop
```

Comment:
changelogs have to start with specific keywords, please use something like `* Changed associtation fetching in bulk edit, to fix infinite loop`